easybuild/easyconfigs/h/HTSlib/HTSlib-1.7-foss-2015b.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'HTSlib'
+version = '1.7'
+
+homepage = "http://www.htslib.org/"
+description = """A C library for reading/writing high-throughput sequencing data.
+ This package includes the utilities bgzip and tabix"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['be3d4e25c256acdd41bebb8a7ad55e89bb18e2fc7fc336124b1e2c82ae8886c6']
+
+# cURL added for S3 support
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('bzip2', '1.0.6'),
+    ('XZ', '5.2.2'),
+    ('cURL', '7.47.1'),
+    ('GCC','4.8.4','',('dummy','')),           
```

Comment:
GCC already part of foss/2015b toolchain. There is no need to include it as dependency. Including it may be confusing and risks listing the wrong version, which is not part of the specified toolchain.